OData/src/System.Web.OData/OData/Query/Expressions/AggregationBinder.cs

```diff
@@ -264,7 +296,7 @@ private Expression BindAccessor(SingleValueNode node)
                     return CreatePropertyAccessExpression(BindAccessor(propAccessNode.Source), propAccessNode.Property);
                 case QueryNodeKind.SingleValueOpenPropertyAccess:
                     var openNode = node as SingleValueOpenPropertyAccessNode;
-                    return Expression.Property(BindAccessor(openNode.Source), openNode.Name);
```

Comment:
I wasn't correct Expression.Property(BindAccessor(openNode.Source), openNode.Name) was used for multiple aggregation transformations like groupby((A,B))/groupby((A)). I modified code to support that case too.